cdapi/urls.py

```diff
@@ -5,7 +5,7 @@
 
 urlpatterns = [  # pylint: disable=invalid-name
     path('admin/', admin.site.urls),
-    re_path('api/(?P<version>(v1|v2))/auth/', include('users.auth_urls')),
+    re_path('api/v1/auth/', include('users.auth_urls')),
```

Comment:
You have to Keep `?P<version>` because it will be used in the views when we migrate to the next versions of the API.